.github/workflows/E2E.yaml

```diff
@@ -1,5 +1,7 @@
 name: End to end tests
 
+run-name: End to end tests - ${{ github.event.inputs.ref != '' && github.event.inputs.ref || github.sha }}
```

Comment:
Not sure what this is supposed to mean 🤔 

And I tested it a bit and now even more confused:
![image](https://github.com/microsoft/AL-Go/assets/43066499/2a9ce5b3-fc2e-43f6-8cfb-9424ef5cc323)

https://github.com/mazhelez/BCApps/actions/runs/9085072498/job/24967509935

Reply:
This construct: ${{ github.event.inputs.ref != '' && github.event.inputs.ref || github.sha }}

returns github.event.inputs.ref if that is not empty - else it returns github.sha
So - it will display the sha on which the tests are run.

Reply:
Problem in your test is that your input variable is called sha - you use mine called ref.
I suggested a change to your repo.

Reply:
True, my bad.